src/EditorFeatures/Core.Wpf/PublicAPI.Unshipped.txt

```diff
@@ -1 +0,0 @@
-
```

Comment:
I know it's boring 🙂but could you keep this file unchanged?

Reply:
Oh interesting, would have though Git find out there's no changes.

Reply:
Oh i see, there were 2 lines inside.